docs/php/activities.md

```diff
@@ -314,8 +314,8 @@ $activityClient->completeExceptionallyByToken($taskToken, new \Error("activity f
 
 ## Local Activity
 
-Temporal has a concept of [Local Activity](/docs/concepts/what-is-a-local-activity).
-To create a Local Activity you should use `#[LocalActivityInterface]` (instead of `#[ActivityInterface]`):
+In Temporal there is a concept of [Local Activity](/docs/concepts/what-is-a-local-activity). To create a local
+Activity you should use `#[LocalActivityInterface]` (instead of `#[ActivityInterface]`):
```

Comment:
Revert this change?

Reply:
It came up as merge conflict, and I seem to have mixed them up.

Reply:
```suggestion
Temporal has a concept of [Local Activity](/docs/concepts/what-is-a-local-activity).
To create a Local Activity you should use `#[LocalActivityInterface]` (instead of `#[ActivityInterface]`):
```

Reply:
```suggestion
Temporal has a concept of [Local Activity](/docs/concepts/what-is-a-local-activity).
To create a Local Activity you should use `#[LocalActivityInterface]` (instead of `#[ActivityInterface]`):
```